communication/slack-config/sig-release/usergroups.yaml

```
@@ -48,36 +50,24 @@ usergroups:
       - release-notes
       - sig-release
     members:
-      - cpanato # SIG Release Technical Lead
-      - jeremyrickard # SIG Release Chair
-      - justaugustus # SIG Release Chair
-      - mehabhalodiya # 1.29 RT Lead Shadow
-      - mickeyboxell # 1.29 RT Lead Shadow
-      - neoaggelos # 1.29 RT Lead Shadow
-      - Priyankasaggu11929 # 1.29 RT Lead
-      - puerco # SIG Release Technical Lead
-      - ramrodo # 1.29 RT Lead Shadow
-      - salaxander # 1.29 Emeritus Adviser
-      - saschagrunert # SIG Release Chair
-      - Verolop # SIG Release Technical Lead
-
-  # Should match SIG Release Leads at all times:
-  # https://git.k8s.io/community/sig-release/README.md#leadership
-  - name: sig-release-leads
-    long_name: SIG Release Leads
-    description: >-
-      SIG Release Leads. Ping for questions on SIG Release process and escalations.
-    channels:
-      - release-ci-signal
-      - release-management
-      - sig-release
-    members:
-      - cpanato # SIG Release Technical Lead
-      - jeremyrickard # SIG Release Chair
-      - justaugustus # SIG Release Chair
-      - puerco # SIG Release Technical Lead
-      - saschagrunert # SIG Release Chair
-      - Verolop # SIG Release Technical Lead
+      - chanieljdan         # 1.32 Docs Lead
+      - cpanato             # SIG Release Technical Lead
+      - drewhagen           # 1.32 Release Signal Lead
+      - fsmunoz             # 1.32 Release Lead
+      - gracenng            # Release Team Subproject Lead
+      - jeremyrickard       # SIG Release Chair
+      - justaugustus        # SIG Release Chair
+      - katcosgrove         # 1.32 EA + Release Team Subproject Lead
+      - mbianchidev         # 1.32 Communications Lead
+      - npolshakova         # 1.32 RT Lead Shadow
+      - puerco              # SIG Release Technical Lead
+      - salehsedghpour      # 1.32 RT Lead Shadow
+      - saschagrunert       # SIG Release Chair
+      - satyampsoni         # 1.32 Release Notes Lead
+      - sreeram-venkitesh   # 1.32 RT Lead Shadow
+      - tjons               # 1.32 Enhancements Lead-
+      - Verolop             # SIG Release Technical Lead
+      - Vyom-Yadav          # 1.32 RT Lead Shadow
```

@fsmunoz, the change needed is following:

that is – update the  `release-team-leads` slack group to remove SIG Release Leads (chair/TLs), and add Grace/Kat (as SIG Release sub-project leads)

```suggestion
      - chanieljdan         # 1.32 Docs Lead
      - drewhagen           # 1.32 Release Signal Lead
      - fsmunoz             # 1.32 Release Lead
      - gracenng            # Release Team Subproject Lead
      - katcosgrove         # 1.32 EA + Release Team Subproject Lead
      - mbianchidev         # 1.32 Communications Lead
      - npolshakova         # 1.32 RT Lead Shadow
      - salehsedghpour      # 1.32 RT Lead Shadow
      - satyampsoni         # 1.32 Release Notes Lead
      - sreeram-venkitesh   # 1.32 RT Lead Shadow
      - tjons               # 1.32 Enhancements Lead
      - Vyom-Yadav          # 1.32 RT Lead Shadow
```

---

Thank you, so it's sort of the opposite option that I knew existed :D